ModelBundle/DataFixtures/MongoDB/LoadDeletedSiteData.php

```diff
@@ -0,0 +1,153 @@
+<?php
+
+namespace OpenOrchestra\ModelBundle\DataFixtures\MongoDB;
+
+use Doctrine\Common\DataFixtures\AbstractFixture;
+use Doctrine\Common\DataFixtures\OrderedFixtureInterface;
+use Doctrine\Common\Persistence\ObjectManager;
+
+use OpenOrchestra\ModelBundle\Document\Content;
+use OpenOrchestra\ModelBundle\Document\ContentAttribute;
+use OpenOrchestra\ModelBundle\Document\Node;
+use OpenOrchestra\ModelInterface\Model\NodeInterface;
+use OpenOrchestra\ModelInterface\DataFixtures\OrchestraFunctionalFixturesInterface;
+
+/**
+ * Class LoadDeletedSiteData
+ */
+class LoadDeletedSiteData extends AbstractFixture implements OrderedFixtureInterface, OrchestraFunctionalFixturesInterface
```

Comment:
As contents and nodes are loaded on different phases (content are loaded before nodes to allow usage of content blocks) I suggest you to split this fixture into two different files, one for content and one for nodes. That way you can have a finer grain on the ``getOrder`` method.